governance.md

```diff
@@ -7,6 +7,20 @@ https://github.com/pangeo-data/governance
 
 ## The Project
 
+The Pangeo Project (The Project) is an open source software project. The goal of The Project is to develop open source software and related technology for the analysis of large scientific datasets. The Project endeavors to extend the broader scientific software ecosystem.
+The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted in public GitHub repositories under the Pangeo-data GitHub organization. Examples of Project Software include the tools and configurations related to the deployment of computational infrastructure. The Services run by the Project consist of public websites and web-services that are hosted under the pangeo.io or pangeo-data.org domains. The Services Examples of Project Services include the Pangeo website (https://pangeo.org and https://pangeo.io), Pangeo-JupyterHub (https://pangeo.pydata.org) and the Pangeo-Binder (https://binder.pangeo.io).
+
+The Project is developed by a team of distributed developers, called Contributors. Contributors are individuals who have contributed code, documentation, designs or other work to one or more Project repositories. Anyone can be a Contributor. Contributors can be affiliated with any legal entity or none. Contributors participate in the project by submitting, reviewing and discussing GitHub Pull Requests and Issues and participating in open and public Project discussions on GitHub, Gitter chat rooms and mailing lists. The foundation of Project participation is openness and transparency.
```

Comment:
Because of the amorphous nature of the project, I think we should consider a broader definition of contributors. (For example, by this definition, @kmpaul is not a contributor, since he has not contributed substantially to the repos.)  It might also be the case that someone contributes substantially to pangeo by working on xarray, which is not formally part of these repos.

How can we adjust the language to account for this?

Reply:
Instead, the poster-boy of the non-committing contributor is @niallrobinson. 😉 

Does our definition of "contributor" encompass Niall's contributions?